crates/snapshot/src/snapshotter.rs

```diff
@@ -79,21 +83,32 @@ impl SnapshotTargets {
 
 impl<DB: Database> Snapshotter<DB> {
     /// Creates a new [Snapshotter].
-    pub fn new(db: DB, chain_spec: Arc<ChainSpec>, block_interval: u64) -> Self {
+    pub fn new(
+        db: DB,
+        snapshots_path: PathBuf,
+        chain_spec: Arc<ChainSpec>,
+        block_interval: u64,
```

Comment:
nit:

I'd like to move snapshots_path + block_interval to a new `SnapshotterConfig` struct